components/window/position.md

```diff
@@ -60,11 +62,17 @@ In this case, the Window will render inside the specified container and not as a
 
 ## Top and Left
 
-The `Top` and `Left` parameters control the Window placement on the page. The resulting position depends on the whole page content and not on the viewport or the current scroll offset.
+The `Top` and `Left` parameters control the Window placement on the page. The resulting position depends on the whole page content and not on the viewport or the current scroll offset. To see the parameters in action, refer to the [example](#example) below.
 
 When the [Window `ContainmentSelector` parameter is set](#containmentselector), the `Top` and `Left` parameters apply with regard to the top-left corner of the containment element.
 
->caption Using Top and Left to manage the Window position
+## Center
+
+ The Telerik Window is automatically centered when the `Top` and `Left` parameters are not set or are explicitly set to `string.Empty`. This ensures the Window appears in the middle of the viewport by default. To see this behavior in action, refer to the [example](#example) below.
```

Comment:
"in the middle of the viewport" is correct only if there is no page scrolling or when the Window is not modal. Otherwise, the Window will appear in the middle of the viewport when the page is not scrolled.